Sources/AppStoreConnectCLI/Commands/TestFlight/BetaTesters/ListBetaTestersCommand.swift

```diff
@@ -84,16 +68,6 @@ struct ListBetaTestersCommand: CommonParsableCommand {
     @Option(help: "Number of included related resources to return.")
     var relatedResourcesLimit: Int?
 
-    func validate() throws {
-        if !filterAppIds.isEmpty && !filterBundleIds.isEmpty {
-            throw ValidationError("Filtering by both Bundle ID and App ID is not supported!")
-        }
-
-        if (!filterAppIds.isEmpty || !filterBundleIds.isEmpty) && !filterGroupNames.isEmpty {
-            throw ValidationError("Only one of these relationship filters ('app-id, bundle-id', 'group-name') can be applied.")
-        }
-    }
```

Comment:
Is there any way that we can keep this validation? (eg. throw Error when user input both identifiers and group names)

Reply:
The user will either enter app id or bundle id. So one id can't be both. So this validation is not required.

Reply:
there is a group name as well `!filterGroupNames.isEmpty`. I remember input both group name and appId is not supported yet by Apple store connect API

Reply:
Thanks for pointing out that. So I will do something like this 
```func validate() throws {
        if (!appLookupOptions.filterIdentifiers.isEmpty) && !filterGroupNames.isEmpty {
            throw ValidationError("Only one of these relationship filters ('app-id/ bundle-id', 'group-name') can be applied.")
        }
    }```

Does this look okay to you?

Reply:
yes this looks good